### ISSUE TITLE
Does it work for Linux vscode?

### ISSUE THREAD

**Issue by YaoLiMuMu** [imlinhanchao/vsc-serialport-helper#11]
Does it work for Linux vscode

**Comment by imlinhanchao** [comment]
I think it can work in Ubuntu. But I haven't device to try it. You can have a try.

**Comment by YaoLiMuMu** [comment]
I don't think so, it doesn't show anything in Serial Port View, and when I refresh serial port list, vscode warning "command 'serialport.refreshEntry' not found"
![image](https://user-images.githubusercontent.com/32826811/146928895-ab0b70f7-1589-46b0-a266-4c0150a7ba08.png)

now I use minicom as my serial print tool

**Comment by imlinhanchao** [comment]
OK. Could you let me know what Linux version you are? I will build virtual machine to debug.

**Comment by YaoLiMuMu** [comment]
It is appreciate for your reply, my linux version is ubuntu 20..04.03, please refer to the picture below
![image](https://user-images.githubusercontent.com/32826811/147631556-40b3c4b0-cb14-44b4-9504-cd6698ea7578.png)

**Comment by imlinhanchao** [comment]
Hi~ Can you try this way to rebuild extension?  https://github.com/imlinhanchao/vsc-serialport-helper/wiki/How-To-Build-SerialPort-Node-module

**Comment by diggit** [comment]
`v0.0.15` works fine on Archlinux. Serial port library rebuild was necessary though.

**Comment by aziz76** [comment]
> 

Hi,

Can you elaborate please? I have tried to rebuild the extensions following the steps [here](https://github.com/imlinhanchao/vsc-serialport-helper/wiki/How-To-Build-SerialPort-Node-module). But I am getting a build error:

`../src/serialport.cpp:486:1: note: in expansion of macro ‘NODE_MODULE’
  486 | NODE_MODULE(serialport, init);
      | ^~~~~~~~~~~
`

I am running Arch Linux
